index.html

```diff
@@ -29,45 +36,54 @@ <h2 class="soft">Software Developer</h2>
             project you need coded, don’t hestiate to contact me.
           </p>
         </div>
-        <img src="./img/Devider.png" alt="" />
-        <div class="bottom">
-          <div class="lang">
-            <h4 class="main">Languages</h4>
-            <ul class="list">
-              <li>Javascript</li>
-              <li>Ruby</li>
-              <li>Html</li>
-              <li>Css</li>
-            </ul>
-          </div>
-          <div class="fwork">
-            <h4 class="main">Framework</h4>
-            <ul class="list">
-              <li>Bootstrap</li>
-              <li>Ruby on Rails</li>
-              <li>RSPec</li>
-              <li>CapyBara</li>
-              <li>Selenium</li>
-            </ul>
-          </div>
+        <div>
+          <img class="devs" src="./img/Devider.png" alt="" />
         </div>
-        <div class="sk">
-          <div class="skills">
-            <h4 class="main">Skills</h4>
-            <ul class="list">
-              <li>Codekit</li>
-              <li>Github</li>
-              <li>Codepen</li>
-              <li>Gitlab</li>
-              <li>Terminal</li>
-            </ul>
-          </div>
-          <div>
-            <!--<h4>Comment out</h4>-->
+        <div class="devi">
+          <img class="vector-4" src="./img/Vector4.png" alt="thin grey line" />
+        </div>
+        <div class="wrapper">
+          <div class="bottom">
+            <div class="lang">
+              <h4 class="main">Languages</h4>
+              <ul class="list">
+                <li class="li-lang">Javascript</li>
+                <li class="li-lang">Ruby</li>
+                <li class="li-lang">Html</li>
+                <li class="li-lang">Css</li>
+              </ul>
+            </div>
+            <div class="fwork">
+              <h4 class="main">Framework</h4>
+              <ul class="list">
+                <li class="li-frame">Bootstrap</li>
+                <li class="li-frame">Ruby on Rails</li>
+                <li class="li-frame">RSPec</li>
+                <li class="li-frame">CapyBara</li>
+                <li class="li-frame">Selenium</li>
+              </ul>
+            </div>
+            <div class="skills">
+              <h4 class="main">Skills</h4>
+              <ul class="list">
+                <li class="li-skills">Codekit</li>
+                <li class="li-skills">Github</li>
+                <li class="li-skills">Codepen</li>
+                <li class="li-skills">Gitlab</li>
+                <li class="li-skills">Terminal</li>
+              </ul>
+            </div>
           </div>
         </div>
-        <img src="./img/Devider.png" alt="" />
+        <img class="devii" src="./img/Devider.png" alt="" />
+        <div class="devi">
+          <img class="vector-4" src="./img/Vector4.png" alt="thin grey line" />
+        </div>
         <div class="socials">
```

Comment:
please try to remove the yellow color when hovering on the links 
![image](https://user-images.githubusercontent.com/61790587/165864156-9e9b30dd-8ee2-4e9f-9718-7aba3544f073.png)